README.md

```diff
@@ -1,3470 +1,188 @@
-config.properties['hazelcast.client.statistics.period.seconds'] = 4;
-```
+```js
+const Client = require('hazelcast-client').Client;
+const Config = require('hazelcast-client').Config;
 
-After enabling the client statistics, you can monitor your clients using Hazelcast Management Center. Please refer to the [Monitoring Clients section](https://docs.hazelcast.org/docs/management-center/latest/manual/html/index.html#monitoring-clients) in the Hazelcast Management Center Reference Manual for more information on the client statistics.
+// Create a configuration object
+const clientConfig = new Config.ClientConfig();
 
-### 7.9.2. Logging Configuration
+// Customize the client configuration
+clientConfig.groupConfig.name = 'cluster-name';
+clientConfig.networkConfig.addresses.push('10.90.0.2:5701');
+clientConfig.networkConfig.addresses.push('10.90.0.3:5701');
 
- By default, Hazelcast Node.js client uses a default logger which logs to the `stdout` with the `INFO` log level. You can change the log level using the `'hazelcast.logging.level'` property of the `ClientConfig.properties`.
- 
-Below is an example of the logging configuration with the `OFF` log level which disables logging.
- 
-```javascript
-cfg.properties['hazelcast.logging.level'] = LogLevel.OFF;
+// Initialize the client with the given configuration
+Client.newHazelcastClient(clientConfig)
+  .then(client => {
+    console.log('Connected to cluster');
+    return client.shutdown();
+  });
 ```
- 
- You can also implement a custom logger depending on your needs. Your custom logger must have `log`, `error`, `warn`, `info`, `debug`, `trace` methods. After implementing it, you can use your custom logger using the `customLogger` property of `ClientConfig`
-  
-See the following for a custom logger example.
- 
-```javascript
-var winstonAdapter = {
-    logger: new (winston.Logger)({
-        transports: [
-            new (winston.transports.Console)()
-        ]
-    }),
-
-    levels: [
-        'error',
-        'warn',
-        'info',
-        'debug',
-        'silly'
-    ],
-
-    log: function (level, objectName, message, furtherInfo) {
-        this.logger.log(this.levels[level], objectName + ': ' + message, furtherInfo);
-    },
-
-    error: function (objectName, message, furtherInfo) {
-        this.log(LogLevel.ERROR, objectName, message, furtherInfo);
-    },
-
-    warn: function (objectName, message, furtherInfo) {
-        this.log(LogLevel.WARN, objectName, message, furtherInfo);
-    },
 
-    info: function (objectName, message, furtherInfo) {
-        this.log(LogLevel.INFO, objectName, message, furtherInfo);
-    },
+You can also configure the client
+[declaratively](https://github.com/hazelcast/hazelcast-nodejs-client#312-declarative-configuration-json) using a JSON file.
 
-    debug: function (objectName, message, furtherInfo) {
-        this.log(LogLevel.DEBUG, objectName, message, furtherInfo);
-    },
+## Features
```

Comment:
Users seem to be interested in various events exposed via client API, so I'd also mention this one in the features list